apps/pie-docs/src/components/text-input/overview/overview.md

```diff
@@ -127,6 +127,11 @@ Use when text input requires clarification of the input format, such as a symbol
 Payment method is only available in trailing content.
 Use when a text input requires a card number to be entered, and reflect the card  entered by the user.
 
+{% notification {
+  type: "warning",
+  message: "This modifier only exists in design, not in development. The functionality is handled by pillar engineers' if required."
```

Comment:
```suggestion
  message: "This modifier only exists in design, not in development. The functionality can be handled by consumers if required."
```

Reply:
i am gonna update the other references too 🤝